src/lib/components/Panel.svelte

```diff
@@ -1,4 +1,8 @@
-<div class="panel" {...$$restProps}>
+<script lang="ts">
+	export let uncentered: boolean
```

Comment:
It's generally a bad idea to use negative words in a module API. Lets rather switch to either `centered`, or, better still (since booleans are also a bad practice), switch to enum, e.g.:
```suggestion
	export let verticalAlign: 'top' | 'center' | 'bottom' = 'center'
```